pkg/loader/github/github.go

```diff
@@ -18,52 +19,63 @@ import (
 	"github.com/gptscript-ai/gptscript/pkg/types"
 )
 
-const (
-	GithubPrefix      = "github.com/"
-	githubRepoURL     = "https://github.com/%s/%s.git"
-	githubDownloadURL = "https://raw.githubusercontent.com/%s/%s/%s/%s"
-	githubCommitURL   = "https://api.github.com/repos/%s/%s/commits/%s"
-)
+type GithubConfig struct {
+	Prefix      string
+	RepoURL     string
+	DownloadURL string
+	CommitURL   string
+	AuthToken   string
+}
 
 var (
-	githubAuthToken = os.Getenv("GITHUB_AUTH_TOKEN")
-	log             = mvl.Package()
+	log                   = mvl.Package()
+	DEFAULT_GITHUB_CONFIG = &GithubConfig{
```

Comment:
This can be unexported: `defaultGithubConfig`. At least, let's not make it all caps so it is consistent with the rest of the codebase.